packages/docs/src/data/items.ts

```diff
@@ -0,0 +1,82 @@
+export default [
+  {
+    img: '/images/Bear.svg',
+    name: 'Bear'
+  },
+  {
+    img: '/images/Butterfly.svg',
+    name: 'Australia'
```

Comment:
The name was left unchanged, should be Butterfly

Reply:
You are right. I have updated the docs again. Thank you~